### PR TITLE
Feature/holiday api

### PR DESCRIPTION
**Pull request by capJuan**

### CODE REVIEW

File: app/api/gov-holiday.js [hunk 1]
```diff
@@ -14,11 +14,24 @@ async function getHolidayCalendarForEngland () {
   }
 }
 
+async function isTodayCustomHoliday () {
+  const url = `${process.env.APPLICATION_API_URI}/holidays/isTodayHoliday`
+  try {
+    await Wreck.get(url)
+    console.error('today is a custom holiday')
```
Comment (govTechSatish) [inline app/api/gov-holiday.js:21]: why this error?

File: test/unit/api/gov-holiday.test.js [hunk 2]
```diff
@@ -1,113 +1,137 @@
...
+// const Holidays = require('../../../app/api/gov-holiday')
+const holidays = require('../../../app/api/gov-holiday')
 
 // Mocking the @hapi/wreck module
-jest.mock('@hapi/wreck', () => ({
-  get: jest.fn()
-}))
+// jest.mock('@hapi/wreck', () => ({
+//   get: jest.fn()
+// }))
```
Comment (govTechSatish) [inline test/unit/api/gov-holiday.test.js:11]: Remove commented code

File: docker-compose.yaml [hunk 3]
```diff
@@ -25,4 +25,4 @@ services:
       RBAC_ENABLED: ${RBAC_ENABLED:-false}
       DATE_OF_TESTING_ENABLED: ${DATE_OF_TESTING_ENABLED:-false}
       ON_HOLD_APP_PROCESS_ENABLED: "true"
-      ON_HOLD_APP_PROCESS_SCHEDULE: "0 0 * * 2-5"
+      ON_HOLD_APP_PROCESS_SCHEDULE: "*/3 * * * *"
```
Comment (govTechSatish) [inline docker-compose.yaml:28]: Don't need to push this.

File: app/api/gov-holiday.js (hunk 1 above)
Comment (capJuan) [reply app/api/gov-holiday.js:21]: Correct this is just a log .

File: test/unit/api/gov-holiday.test.js (hunk 2 above)
Comment (capJuan) [reply test/unit/api/gov-holiday.test.js:11]: well spot thanks

File: docker-compose.yaml (hunk 3 above)
Comment (capJuan) [reply docker-compose.yaml:28]: ok done

**Review by govTechSatish — APPROVED** [review]

**Pull request merged**